www/include/configuration/configServers/formServers.ihtml

```diff
@@ -1,136 +1,138 @@
+        <tr class="list_two">
             <td class="FormRowField"><img class="helpTooltip" name="remote_additional_id"> {$form.remote_additional_id.label}</td>
             <td class="FormRowValue">{$form.remote_additional_id.html}</td>
         </tr>
-		{/if}
-    {if $form.header.Remote_Configuration.label}
-    <tr class="list_lvl_1">
-      <td class="ListColLvl1_name" colspan="2">
-          <h4>{$form.header.Remote_Configuration}</h4>
-      </td>
-      <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="http_method">{$form.http_method.label}</td><td class="FormRowValue">{$form.http_method.html}</td></tr>
-      <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="http_port">{$form.http_port.label}</td><td class="FormRowValue">{$form.http_port.html}</td></tr>
-      <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="no_check_certificate">{$form.no_check_certificate.label}</td><td class="FormRowValue">{$form.no_check_certificate.html}</td></tr>
-      <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="no_proxy">{$form.no_proxy.label}</td><td class="FormRowValue">{$form.no_proxy.html}</td></tr>
-    </tr>
-    {/if}
-		<tr class="list_lvl_1">
-      <td class="ListColLvl1_name" colspan="2">
-        <h4>{$form.header.gorgone_Informations}</h4>
-      </td>
-    </tr>
-		<tr class="list_one">
-            <td class="FormRowField"><img class="helpTooltip" name="gorgone_communication_type"> {$form.gorgone_communication_type.label}</td>
-            <td class="FormRowValue">{$form.gorgone_communication_type.html}</td>
+        {/if}
+        {if $form.header.Remote_Configuration.label}
+        <tr class="list_lvl_1">
+            <td class="ListColLvl1_name" colspan="2">
+                <h4>{$form.header.Remote_Configuration}</h4>
+            </td>
+            <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="http_method">{$form.http_method.label}</td><td class="FormRowValue">{$form.http_method.html}</td></tr>
+            <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="http_port">{$form.http_port
+                .label}</td><td class="FormRowValue">{$form.http_port.html}</td></tr>
+            <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="no_check_certificate">{$form.no_check_certificate.label}</td><td class="FormRowValue">{$form.no_check_certificate.html}</td></tr>
+            <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="no_proxy">{$form.no_proxy.label}</td><td class="FormRowValue">{$form.no_proxy.html}</td></tr>
         </tr>
-		<tr class="list_two">
-            <td class="FormRowField"><img class="helpTooltip" name="gorgone_port"> {$form.gorgone_port.label}</td>
-            <td class="FormRowValue">{$form.gorgone_port.html}</td>
+        {/if}
+        <!-- specific gorgone fields for remote or poller -->
```

Comment:
only lines 45 to 65 were modified.
other lines are indentation correction and list_one/list_two consistency